Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+def conda_dir = "~/.miniconda_covid_model_infections"
+def env_name="infections-$BUILD_NUMBER"
+
+def cloneRepoToBuild(code_branch) {
+  sh "echo \$(hostname)"
+  sh "pwd"
+  sh "echo 'mkdir $BUILD_NUMBER'"
+  sh "mkdir $BUILD_NUMBER"
+  sshagent (credentials: ['svccovidvi-privatekey']) {
+  // sshagent (credentials: ['jenkins-general']) {
+    sh "echo 'Downloading source code...'"
+    sh "git clone --branch $code_branch git@github.com:ihmeuw/covid-model-infections.git $BUILD_NUMBER/covid-model-infections/"
+    sh "ls -lr $BUILD_NUMBER"
+    sh "echo 'Source code downloaded'"
+  }
+}
+
+pipeline {
+  //The Jenkinsfile of ssh://git@stash.ihme.washington.edu:7999/scic/covid-snapshot-etl-orchestration.git
```

Comment:
You should store this in a stash repo as they did with the snapshot/etl orchaestration.  It contains a lot of internal info